### PR TITLE
francois's suggestion to update data

### PR DESCRIPTION
**Pull request by IssamT**

### CODE REVIEW

File: src/RVRP_data.jl [hunk 1]
```diff
@@ -1,118 +1,81 @@
...
+end
+
+struct RvrpProblem
+    problem_id::String
+    problem_type::ProblemType
+    vehicles::Vector{VehicleType}
+    distance_matrix::Array{Float64,2}
+    travel_times_matrix::Array{Float64,2}
+    # Requests
+    pickups::Vector{Request}
```
Comment (IssamT) [inline src/RVRP_data.jl]: @francois-DV In your suggestion I think we should remove the vectors pickups, deliveries, shipments. Right?

File: src/RVRP_data.jl [hunk 2]
```diff
@@ -1,118 +1,81 @@
...
+    deliveries::Vector{Request}
+    operations::Vector{Request}
+    shipments::Vector{Request}
+    picked_shipments::Vector{Request}
```
Comment (IssamT) [inline src/RVRP_data.jl]: How do we handle ongoing shipments?

File: src/RVRP_data.jl [hunk 3]
```diff
@@ -1,118 +1,81 @@
...
+
+struct Request
+    id::String
+    index::Int
+    operations::Vector{Operation} # Sequence of operations ; can be limited to one; can be a pair of Pickup and Delivery, or a triplets including a cleaning first, etc
+end
+
+struct VehicleBaseType
+    prices::ResourcePrices
+    capacity::ResourceConsumptions
```
Comment (IssamT) [inline src/RVRP_data.jl]: It is not intuitive to put capacity under the type ResourceConsumptions

File: src/RVRP_data.jl [hunk 4]
```diff
@@ -1,118 +1,81 @@
...
+end
+
+struct Depot <: AbstractNode
+    location::Location
+    time_windows::Vector{TimeWindow} # optional
+end
+
+struct Operation <: AbstractNode
+    location::Location
+    conso::ResourceConsumptions
```
[the remark this answers is not in the document]
Comment (vitornesello) [reply src/RVRP_data.jl]: we dont need to model the preemption aspect like this in the data structure. this is done automatically by the interface with coluna

File: src/RVRP_data.jl (hunk 2 above)
Comment (vitornesello) [reply src/RVRP_data.jl]: # Requests that are ongoing, where the Int part represents the
    # number of finished operations in the given request:
    ongoing_operations::Vector{Pair{Request,Int}}

File: src/RVRP_data.jl (hunk 1 above)
Comment (vitornesello) [reply src/RVRP_data.jl]: i think these are still valid for the sake of facilitating the access

File: src/RVRP_data.jl (hunk 3 above)
Comment (vitornesello) [reply src/RVRP_data.jl]: we can have this: `const ResourceValues = Dict{RESOURCE,Float64}` and use the name of the field of each structure to say whether it is a price, a capacity, an initial state or a consumption

**Pull request closed without merging**